src/Validator/Constraints/IsTrueValidator.php

```diff
@@ -37,12 +38,11 @@ class IsTrueValidator extends ConstraintValidator
      *
      * @var bool
      */
-    protected $verifyHost;
+    protected bool $verifyHost;
```

Comment:
Typed properties are supported by PHP 7.4+
You need to remove `bool` to keep compatibility with lower PHP version